py/selenium/webdriver/firefox/firefox_profile.py

```diff
@@ -53,13 +53,13 @@ def __init__(self, profile_directory=None):
            This defaults to None and will create a new
            directory when object is created.
         """
+        self._strict_timestamps = False
```

Comment:
This isn't a backwards compatible change, and this is the first time someone has reported an issue. Let's keep current behavior and allow users to toggle it if it causes them a problem.

Reply:
I reverted the change back to the default value of False. As I described in the bug comments, I believe that the client code should work the same for both True/False values for dates after 1980.

Reply:
Ok, I got bad information about this change, and then didn't look deeper (and/or read the conversation in that issue carefully). I was given the impression that the change meant it wasn't guaranteeing time stamp information, which appears not to be the case.

So, the only non-backwards compatible concern is if someone for some reason is relying on the fact that they will get an exception if files before 1980 are being zipped. That does not seem like a reasonable concern for Selenium users. In which case even the toggle is excessive. So. Yeah, let's remove the toggle and just set to true. 

Thanks, and sorry for the extra hassle.

Reply:
> let's remove the toggle and just set to true.

i guess you mean false in

```py
zipfile.ZipFile(fp, "w", zipfile.ZIP_DEFLATED, strict_timestamps=False)
```

> this is the first time someone has reported an issue

this happens when using firefox from the [nix](https://github.com/NixOS/nix) store
where all files have timestamp zero = 1970-01-01

Reply:
Yes, `False`, and makes sense